src/utils/index.js

```diff
@@ -53,5 +58,31 @@ export const formatDecimal = (decimal, significantDecimalPlaces = 2) => {
     return `${integers}.${adjustedDecimals}`;
 };
 
+export const getImageByAccountType = (type) => {
+    switch (type) {
+        case PORTFOLIO_SOURCE.ETHEREUM_WALLET: {
+            return ethLogo;
+        }
+        case PORTFOLIO_SOURCE.QUANTUM_WALLET: {
+            return qtumLogo;
+        }
+        case PORTFOLIO_SOURCE.LOOPRING: {
+            return lrcLogo;
+        }
+        case PORTFOLIO_SOURCE.BINANCE: {
+            return binanceLogo;
+        }
+        default: {
+            throw new Error(`unhandled field type ${type} given`);
+        }
+    }
+};
+
+export const getQtumFromSatoshis = (satoshis) =>
```

Comment:
This could easily be used for a future BTC wallet implementation too. Why not rename this to `btcFromSatoshis` (since QTUM takes a lot of naming conventions from Bitcoin too)? 
Plus, it's not a good idea to work with native `Number` on JS. It would be a lot better to return a `Decimal` instance instead.

Reply:
The fuction already returns a `Decimals`, since the `getQuantumPortfolio` works with `Decimals`.